src/operator/nn/deconvolution.cu

```diff
@@ -40,9 +40,15 @@ static CuDNNDeconvolutionOp<DType> &GetCuDNNDeconvOp(const DeconvolutionParam& p
                                                      const std::vector<TShape>& in_shape,
                                                      const std::vector<TShape>& out_shape,
                                                      const RunContext& rctx) {
+#if DMLC_CXX11_THREAD_LOCAL
```

Comment:
What about `dmlc::ThreadLocalStore`?

Reply:
not entirely sure. but the way we are using right now is to use MX_THREAD_LOCAL and thread_local depending on the macro.